implementations/rust/ockam/ockam_node/src/messages.rs

```diff
@@ -30,7 +43,16 @@ pub enum NodeMessage {
     /// Stop an existing worker
     StopWorker(Address, bool, SmallSender<NodeReplyResult>),
     /// Start a new processor
-    StartProcessor(Address, SenderPair, SmallSender<NodeReplyResult>),
+    StartProcessor {
+        /// The set of addresses in use by this processor
+        addrs: Vec<Address>,
```

Comment:
Let's put metadata along with the Address, so you don't need to find the correspondence later

Reply:
yeah, sounds cleaner